client.py

```diff
@@ -0,0 +1,83 @@
+#! /usr/bin/env python
+
+# Copyright (c) 2019 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from __future__ import print_function, absolute_import
+
+
+from framework import patch_environment
+patch_environment()
+
+
+def main():
+    import sgtk
+    import json
+
+    from python import desktop_client
+
+    user = sgtk.authentication.ShotgunAuthenticator().get_default_user()
+    if not user:
+        print("Unable to create a Desktop Client unauthenticated.")
+        return 1
+
+    sgtk.set_authenticated_user(user)
+
+    client = desktop_client.DesktopClient(user.create_sg_connection())
+    commands = client.call_server_method("list_supported_commands")
```

Comment:
`list_supported_commands`, I think that's a Shotgun Create only thing, right?

Reply:
yes